app/main/posts/common/post-actions.directive.js

```diff
@@ -71,7 +71,7 @@ function PostActionsDirective(
                 return;
             }
 
-            if ($routeParams.view !== 'data' && $location.path().indexOf('data') === -1) {
+            if ($stateParams.view !== 'data' && $location.path().indexOf('data') === -1) {
```

Comment:
this needs to go out and be replaced with non-deprecated $uirouterglobal or similar